keyboards/hotdox/keymaps/bliss/readme.md

```diff
@@ -0,0 +1,8 @@
+# Bliss for Hotdox
+
+This keymap for the ergodox hotdox is a take on a developpement setup I quite
+enjoy. Developpers constantly use brackets and they should therefore be
+accessible instead of hidden away behind a shift key. Also this setup aims to minimize hand travel with a layer containing a numpad as well as arrows. Finally,
+this setup incorporates a dymamic macro recorder for an easy, software agnostic way to program macros.
```

Comment:
Spelling corrections.

```suggestion
This keymap for the ergodox hotdox is a take on a development setup I quite
enjoy. Developers constantly use brackets and they should therefore be
accessible instead of hidden away behind a shift key. Also this setup aims to minimize hand travel with a layer containing a numpad as well as arrows. Finally,
this setup incorporates a dynamic macro recorder for an easy, software agnostic way to program macros.
```